### PR TITLE
add badges, and center image

### PR DESCRIPTION
**Pull request by felipedotcom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
-Welcome
+![CyBear Jinni](https://user-images.githubusercontent.com/9304740/94843279-24a49900-0425-11eb-83f3-87e8ba40b1dc.png)
+
+![dart CI](https://github.com/CyBear-Jinni/CBJ_Smart-Device/workflows/Dart%20CI/badge.svg)
+
+[![style: effective dart](https://img.shields.io/badge/style-effective_dart-40c4ff.svg)](https://pub.dev/packages/effective_dart)
+
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+
+[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://github.com/ellerbrock/open-source-badges/)
```
Comment (guyluz11) [inline README.md:11]: All the badges should be in the same line

File: README.md [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+
+Welcome!
```
Comment (guyluz11) [inline README.md:13]: Make bigger

File: README.md (hunk 1 above)
Comment (guyluz11) [reply README.md:11]: Open source badge should not link to the github of the badge.
You can link it to the [wikipedia](https://en.wikipedia.org/wiki/Open_source) page of it or get creative.

**Review by guyluz11 — COMMENTED** [review]
Please fix

File: README.md (hunk 2 above)
Comment (guyluz11) [reply README.md:13]: All the badges should be in the same line and centered

File: README.md [hunk 3]
```diff
@@ -2,15 +2,12 @@
 
 ![CyBear Jinni](https://user-images.githubusercontent.com/9304740/94843279-24a49900-0425-11eb-83f3-87e8ba40b1dc.png)
 
-![dart CI](https://github.com/CyBear-Jinni/CBJ_Smart-Device/workflows/Dart%20CI/badge.svg)
+<div align="center">
 
-[![style: effective dart](https://img.shields.io/badge/style-effective_dart-40c4ff.svg)](https://pub.dev/packages/effective_dart)
+![dart CI](https://github.com/CyBear-Jinni/CBJ_Smart-Device/workflows/Dart%20CI/badge.svg)[![style: effective dart](https://img.shields.io/badge/style-effective_dart-40c4ff.svg)](https://pub.dev/packages/effective_dart)[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)[![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.png?v=103)](https://en.wikipedia.org/wiki/Open_source)
+</div>
```
[the remark this answers is not in the document]
Comment (guyluz11) [reply README.md:8]: And please remove the one about `effective dart` as we are going to move to `lint` instead.

[the remark this answers is not in the document]
Comment (guyluz11) [reply README.md:8]: If you can do one similar to the `style (effective dart)` but with lint `style (lint)` that could be great.
And make it point to [here ](https://pub.dev/packages/lint/example)

File: README.md (hunk 1 above)
Comment (felipedotcom) [reply README.md]: sure

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (felipedotcom) [reply README.md:8]: sure

File: README.md [hunk 4]
```diff
@@ -2,15 +2,12 @@
 
 ![CyBear Jinni](https://user-images.githubusercontent.com/9304740/94843279-24a49900-0425-11eb-83f3-87e8ba40b1dc.png)
```
[the remark this answers is not in the document]
Comment (felipedotcom) [reply README.md]: ok

**Comment by guyluz11** [comment]
That's looks awesome.

Please change `https://pub.dev/packages/lint/example` link to `https://pub.dev/packages/lint`

I am closing this pull request as it is open to master and we do not merging straight to master.

Please open it again to dev with the small fix and I will merge it.

Thanks

**Comment by felipedotcom** [comment]
done

**Pull request closed without merging**